src/Features/Core/Shared/Utilities/LinkedFilesSymbolEquivalenceComparer.cs

```diff
@@ -28,16 +28,12 @@ public LinkedFilesSymbolEquivalenceComparer(SymbolEquivalenceComparer symbolEqui
 
         bool IEqualityComparer<ISymbol>.Equals(ISymbol x, ISymbol y)
         {
-            return x.Kind == y.Kind && (x.IsKind(SymbolKind.Local) || x.IsKind(SymbolKind.Label) || x.IsKind(SymbolKind.RangeVariable))
-                 ? x.Name == y.Name
-                 : _symbolEquivalenceComparer.Equals(x, y);
```

Comment:
`_symbolEquivalenceComparer` is now unused.  If we remove it, we can remove the constructor and the `IgnoreAssembliesInstance`.